### PR TITLE
Apposite 20487 d mod08 lm

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Comments inline.

### CODE REVIEW

File: Instructions/20487D_MOD08_LAB_MANUAL.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
+1. Open the command prompt, and then navigate to **[Repository Root]\AllFiles\Mod08\Labfiles\Lab1\Blueyonder.Service** folder.
 2. Run the project.
 
 #### Task 2: Record .NET ETW events in PerfView
 
-1. Open **PerfView** and collect all the events
+   Open **PerfView**, and then collect all the events
 
-#### Task 3: Run a script to invoke service and meck it to throw exceptions
+#### Task 3: Run a script to invoke the service and meck it to throw exceptions
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAB_MANUAL.md]: Should this be "make" instead of "meck"?

File: Instructions/20487D_MOD08_LAB_MANUAL.md [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
 
-1. Open **Command Line** and navigate to **[Repository Root]\AllFiles\Mod08\Labfiles\Lab1\Blueyonder.Service** folder.
-2. Run the project in **docker** on port **8080:80** and named **myapp**.
+1. Open the command prompt, and then navigate to the **[Repository Root]\AllFiles\Mod08\Labfiles\Lab1\Blueyonder.Service** folder.
+2. Run the project in docker on port **8080:80** and name **myapp**.
 
-#### Task 2: Record LTTng events with the lttng CLI tool
+#### Task 2: Record LTTng events with the Lttng CLI tool
 
 1. Enter to **myapp** container the shell.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAB_MANUAL.md]: Instruction not clear.

File: Instructions/20487D_MOD08_LAB_MANUAL.md [hunk 3]
```diff
@@ -80,69 +80,69 @@ In this lab, you will use Application Insights to monitor and diagnose a web ser
...
+- Analyze performance by using Application Insights.
 
 ### Exercise 1: Add the Application Insights SDK
 
 #### Task 1: Add the Application Insights SDK to the web service project
 
 1. Open **Azure Portal**.
-2. Setup the **Application Insights** for the new create app service named **blueyondermod08lab2**{YourInitials}.
+2. Setup **Application Insights** for the new **blueyondermod08lab2***{YourInitials}* app service.
  - In the setup config the **Runtime/Framework** to **ASP.NET Core**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAB_MANUAL.md]: Instruction not clear. Should it be, "In the setup, configure **Runtime/Framework** to **ASP.NET Core**."?

**Comment by barucht** [comment]
closing this PR
the editing was done on an unfinished version.

**Pull request merged**